code-studio/src/js/assessments/levelGroup.js

```diff
@@ -1,6 +1,8 @@
 /* global appOptions, Dialog */
 
-function initLevelGroup(
+window.levelGroup = {levels: {}};
+
+window.initLevelGroup = function (
```

Comment:
Brent and I have been trying to namespace anything we move over onto `window.dashboard`, or a similar space like `window.levelbuilder` to avoid putting any bare functions on window.